src/components/grid-aware/ImageHeader/ImageHeader.tsx

```diff
@@ -28,9 +30,13 @@ const ImageHeader = ({
     <div className={s.gridAreaText}>
       <h1 className={s.title}>{title}</h1>
       <div className={s.subtitle}>{subtitle}</div>
-      <div className={s.description}>{description}</div>
+      <div className={s.description}>
+        {description || (
+          <div className={s.descriptionWrapper}>{descriptionNode}</div>
```

Comment:
I think we should be able to collapse `s.descriptionWrapper` into `s.description`, especially if you merge the `description` and `descriptionNode` props into one. Were there any significant styling differences between the string-only version and the non-string version?